### ISSUE TITLE
Firefox NVDA not announcing label file input type

### ISSUE THREAD

**Issue by joe-watkins** [nvaccess/nvda#9953]
### Steps to reproduce:
1. Visit https://s.codepen.io/joe-watkins/debug/QewRrQ with Firefox and NVDA.
2. Tab to the input with file type.

### Actual behavior:
User tabs to input with `file` type and label associated with field is not announced.

*NVDA does announce accessible name when arrow keys are used for navigation.
*Also noticed `aria-describedby` is not announced.

### Expected behavior:
When focus moves to the input with `file` type via tab navigation the accessible name for the input is announced.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA 2018.1.1

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
Firefox 68.0.1

#### Other information about your system:
n/a

### Other questions
n/a

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

**Comment by DrSooom** [comment]
@joe-watkins: Please try to reproduce this with the latest [beta of NVDA (beta-18133)](https://ci.appveyor.com/api/buildjobs/t7pflc1win8kdioo/artifacts/output/nvda_snapshot_beta-18133,b29acb42.exe). You don't have to install it, just run the Installer. Thanks.

**Comment by charmarkk** [comment]
I'm hearing the same thing I was the other day (to which @joe-watkins filed this issue for us). Tabbing to the button leads to a readout of `clickable browse, button, no file selected`. Using arrow keys we get `clickable i am the label, button, browse, no file selected`.

Thanks for putting the codepen together Joe!

**Comment by Adriani90** [comment]
I cannot reproduce this with NVDA alpha 19256,3cd482f5 and Firefox 70. The field label "text field here" is properly announced when using tab in focus mode and browse mode and also when using arrow keys in browse mode.
However, when pressing tab from the edit field it says "I am the label grouping". I guess this should be announced on the edit field itself and not on the "browse" button.
cc: @jcsteh any thoughts?

**Comment by jcsteh** [comment]
> I cannot reproduce this with NVDA alpha 19256,3cd482f5 and Firefox 70. The field label "text field here" is properly announced when using tab in focus mode and browse mode and also when using arrow keys in browse mode.

This issue with file inputs was fixed in Firefox 70. See https://bugzilla.mozilla.org/show_bug.cgi?id=1316965

> However, when pressing tab from the edit field it says "I am the label grouping". I guess this should be announced on the edit field itself and not on the "browse" button.

No, because this label is associated with the file input, not the edit field.
